nvchecker/archlinux-proaudio.toml

```diff
@@ -263,6 +263,12 @@ github = "swh/lv2"
 use_max_tag = true
 prefix = "v"
 
+[triceratops]
+source = "regex"
+url = "https://sourceforge.net/projects/triceratops/files/"
```

Comment:
Cool if this works reliably, but did you try using the rss similar to the tuxguitar section below?